[master<20230420] Improve backup and restore docs (#828)
memgraph/docs: memgraph_versioned_docs/version-2.5.0/how-to-guides/create-backup.md

data in a recoverable format. On startup, it searches for previously saved
durability files and uses them to recreate the most recent DB state.

> When talking about the data directory, we are actually talking about two

Author: ```suggestion
When talking about the data directory in the context of backup and restore, we are actually talking about two
```

### 1. Lock the data directory

> To disable file deletions in the data directory, run the following query in

Author: ```suggestion
To disable deletions of snapshot or WAL files, run the following query in
```
